middle.mkd

```diff
@@ -274,8 +274,9 @@ A GeoJSON object with the type "Feature" is a feature object.
 A GeoJSON object with the type "FeatureCollection" is a feature
 collection object. A feature collection object MUST have a member with
 the name "features". The value of "features" is an array. Each element
-of the array is a feature object as defined above. It is possible for
-this array to be empty.
+of the array MUST be a feature object as defined above. It is possible
+for this array to be empty. The values in the array are ordered, meaning
+that GeoJSON implementations MUST maintain order when processing them.
```

Comment:
Are you saying that a processor is defined as an entity that slurps up GeoJSON and spits out the same GeoJSON? I'm not even sure what that's good for. 

Or are you saying that a processor is something that converts GeoJSON to some other representation that is supposed to be equivalent? Then it would seem to be obvious that if it sees an array with JSON semantics that it turns that into whatever an array looks like in the other representation.

I content that we can talk about the GeoJSON object and fully and unambiguously describe it without reference to a processing model.